src/messageComposer/middleware/pollComposer/state.ts

```diff
@@ -35,7 +35,7 @@ export const pollStateChangeValidators: Partial<
     if (!numericMatch && value) {
       return { max_votes_allowed: 'Only numbers are allowed' };
     }
-    if (value?.length > 1 && !value.match(VALID_MAX_VOTES_VALUE_REGEX))
+    if (!value.match(VALID_MAX_VOTES_VALUE_REGEX))
```

Comment:
It should not be validation on change, but rather on blur. I think it is too early to validate when you start typing 10.

Reply:
Makes sense. We had this for `onChange` on react native, so I thought it would be a bug that way. But what you are saying makes sense. I will close it in that case. Thanks :)

Reply:
and on blur this validation happens

Reply:
Yes, it seems to happen on blur